libs/vkd3d/acceleration_structure.c

```diff
@@ -408,12 +409,14 @@ void vkd3d_acceleration_structure_emit_postbuild_info(
 
     for (i = 0; i < count; i++)
     {
-        vk_acceleration_structure = vkd3d_va_map_place_acceleration_structure(
-                &list->device->memory_allocator.va_map, list->device, addresses[i]);
+        vkd3d_va_map_try_read_rtas(&list->device->memory_allocator.va_map,
```

Comment:
This is broken. If application records an RTAS copy command before the build (it is valid if submitted out of order), things will break. If no object is found, you'll need to explicitly place an RTAS (as it's the common case) and use that. Flag a FIXME in this case, since this is one of the reasons EXT_omm is kinda broken as-is.